etc/omero.properties

```diff
@@ -819,15 +819,19 @@ omero.client.scripts_to_ignore=/omero/figure_scripts/Movie_Figure.py,\
 /omero/figure_scripts/ROI_Split_Figure.py,/omero/export_scripts/Make_Movie.py,\
 /omero/setup_scripts/FLIM_initialise.py,/omero/import_scripts/Populate_ROI.py
 
+# Flag to show/hide "Orphaned images" container. Only accept "true" or "false"
+omero.client.ui.tree.orphans.enabled=true
 # Name of the "Orphaned images" container located in client tree data manager.
-omero.client.ui.tree.orphans.name=Orphaned images
-
+omero.client.ui.tree.orphans.name=Orphaned Images
 # Description of the "Orphaned images" container.
 omero.client.ui.tree.orphans.description=This is a virtual container with orphaned images. These images are not linked anywhere. Just drag them to the selected container.
 
+# helper combining all orphans properties. Do not set manually that property.
+omero.client.ui.tree.orphans={"enabled": "${omero.client.ui.tree.orphans.enabled}", "name": "${omero.client.ui.tree.orphans.name}", "description": "${omero.client.ui.tree.orphans.description}"}
```

Comment:
If I understand correctly reading throught this PR and the previous one, this property represents a configuration node and its principal function is to define the configuration leaves attached to it. Assuming this configuration is never meant to be set, would it be worth blacklisting it from the generated configuration page in https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/install/config_parser.py#L167.